install-sddm-theme.sh

```
@@ -242,6 +266,22 @@ function skipLoadingAnimation {
   done
 }
 
+function compileShaders {
+    if [ ! -f components/doorShader.frag.qsb ]; then
+        echo "Compiling shaders..."
+        if [[ ! -f "$components/doorShader.frag.qsb" && -z "$RECOMPILE_SHADERS" ]]; then  
```

Shellcheck (0.10.0)

[warning] components is referenced but not assigned.

(SC2154)